### PR TITLE
Added more tests to increase coverage

### PR DESCRIPTION
**Pull request by dmalarev**
There are two missing branches in the RouteGenerator and PlayState class
which aren't covered yet.
The reason why is because the RouteGenerator uses a random generator in
the middle of the method that is difficult to mock, while PlayState uses
the VRApplication, which causes a nullpointerException if I try to use
it.
Overall, the branch coverage is now around 79.8%

### CODE REVIEW

**Comment by dalderliesten** [comment]
Good use of mocking, looks good to me.

**Comment by dmalarev** [comment]
fixing checkstyle. give me a minute

**Comment by dalderliesten** [comment]
Which error is it throwing?

**Comment by dmalarev** [comment]
just some missing javadoc and unused imports. already fixed, will commit now

File: src/test/java/nl/tudelft/pixelperfect/client/ServerListenerTest.java [hunk 1]
```diff
@@ -119,4 +126,12 @@ public void testCompletedEventMessageReceived() {
     verify(mockedLog).complete(0);
   }
```
Comment (dalderliesten) [inline src/test/java/nl/tudelft/pixelperfect/client/ServerListenerTest.java]: Javadoc.

**Comment by FlorisXll** [comment]
LGTM

**Comment by dmalarev** [comment]
everything should be ready now

**Pull request merged**